source/whats-new.txt

```diff
@@ -22,6 +23,28 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.7:
+
+What's New in 4.7
+-----------------
+
+New features of the 4.7 {+driver-short+} release include:
+
+- Running ``MongoClient.connect()`` is now optional when connecting to your MongoDB instance
+- Use Zstandard compression when communicating with your MongoDB instance
+- Add support for the ``maxConnecting`` connection option
+- Change stream documents now show your updated documents before and after MongoDB applied changes to them
+- Added support for new change stream fields related to cluster to cluster replication
+- The ``estimatedDocumentCount`` method now uses the ``count`` database command
```

Comment:
```suggestion
- The ``estimatedDocumentCount()`` method now uses the ``$count`` database command 
```
Suggestion: Link to the server page about count, or if we have a page here about how to use count/estimatedDocumentCount()

Reply:
Removed trailing space and adding suggestion